tests/integration_tests/test_trainer.py

```diff
@@ -173,3 +183,58 @@ def test_changing_parameters_on_plateau(tmpdir):
     model = LudwigModel(config, backend=LocalTestBackend())
 
     model.train(training_set=data_csv, validation_set=val_csv, test_set=test_csv, output_directory=tmpdir)
+
+
+@pytest.mark.distributed
+def test_lightgbm_dataset_partition(ray_cluster_2cpu):
+    # Create a LightGBM model with a Ray backend
+    config = {
+        "input_features": [{"name": "in_column", "type": "binary"}],
+        "output_features": [{"name": "out_column", "type": "binary"}],
+        "model_type": "gbm",
+        LIGHTGBM_TRAINER: {"epochs": 1, "batch_size": 128},
```

Comment:
```suggestion
```

Suggesting to remove this from the config here since it doesn't seem to be used. 

Also heads up that parameters for any trainer type live on the top-level `trainer` section of the config (not `LIGHTGBM_TRAINER`). For GBM models, `epochs` and `batch_size` are not defined. Length of training can be controlled with `num_boost_round`.

Reply:
Ah, good catch!